ruby/lib/core/sender.rb

```diff
@@ -29,10 +29,17 @@ class Sender < Link
 
     # Open the {Sender} link
     #
+    # @note Sender **auto-settlement** only occurs for a delivery after the
+    #     sender receives a settled disposition for that delivery. Otherwise,
+    #     there would be no way to receive any further events for that delivery
+    #     (such as the subsequent on_settle message that might be expected when
+    #     the receiver finally settles the message).
```

Comment:
I honestly don't understand why auto-settlement is called out specially here (or above). There's nothing special about it. Its described below under its option.

Reply:
Deleted.